docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,456 @@
++
+<4> If the snapshot was successful.
+
+. The snapshot `snapshot-20200617` contains the two indices we want to restore.
+You might have multiple snapshots from which you could restore the target indices. Choose the latest snapshot.
++
+. Now that we found a snapshot, we will proceed with the data stream preparation for restoring the lost data. We will
+check the index metadata to see if any index is part of a data stream:
++
+[source,console]
+----
+GET kibana_sample_data_flights,.ds-my-data-stream-2022.06.17-000001?features=settings&flat_settings
+----
+// TEST[skip:illustration purposes only]
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  ".ds-my-data-stream-2022.06.17-000001" : {                                <1>
+    "aliases" : { },
+    "mappings" : { },
+    "settings" : {                                                          <2>
+      "index.creation_date" : "1658406121699",
+      "index.hidden" : "true",
+      "index.lifecycle.name" : "my-lifecycle-policy",
+      "index.number_of_replicas" : "1",
+      "index.number_of_shards" : "1",
+      "index.provided_name" : ".ds-my-data-stream-2022.06.17-000001",
+      "index.routing.allocation.include._tier_preference" : "data_hot",
+      "index.uuid" : "HmlFXp6VSu2XbQ-O3hVrwQ",
+      "index.version.created" : "8020299"
+    },
+    "data_stream" : "my-data-stream"                                        <3>
+  },
+  "kibana_sample_data_flights" : {                                          <4>
+    "aliases" : { },
+    "mappings" : { },
+    "settings" : {
+      "index.creation_date" : "1655121541454",
+      "index.number_of_replicas" : "0",
+      "index.number_of_shards" : "1",
+      "index.provided_name" : "kibana_sample_data_flights",
+      "index.routing.allocation.include._tier_preference" : "data_content",
+      "index.uuid" : "jMOlwKPPSzSraeeBWyuoDA",
+      "index.version.created" : "8020299"
+    }
+  }
+}
+----
+// TEST[skip:illustration purposes only]
++
+<1> The name of an index.
++
+<2> The settings of this index that contains the metadata we are looking for.
++
+<3> This indicates that this index is part of a data stream and displays the data stream name.
++
+<4> The name of the other index we requested.
++
+The response above shows that `kibana_sample_data_flights` is not part of a data stream because it doesn't have a
+field called `data_stream` in the settings.
++
+On the contrary, `.ds-my-data-stream-2022.06.17-000001` is part of the data stream called `my-data-stream`. When you
+find an index like this, which belongs to a data stream, you need to check if data are still being indexed.
+You can see that by checking the `settings`, if you can find this property: `"index.lifecycle.indexing_complete" : "true"`,
+it means that indexing is completed in this index and you can continue to the next step.
++
+If it is not there you need to rollover the data stream so you can restore the missing data without blocking
```

Comment:
```suggestion
```suggestion
If `index.lifecycle.indexing_complete` is not there or is configured to `false` you need to rollover the data stream so you can restore the missing data without blocking
```
```